### ISSUE TITLE
Hi there! Quick question for you

### ISSUE THREAD

**Issue by unphased** [ldx/dotfiles#1]
I noticed that you have vim-fugitive and the vim indentfinder plugin. 

I am in the same boat, and basically the problem I have is that whenever I try to run fugitive's `:Gdiff` command, I get a massive error caused by some indentfinder routine, and it just totally drops the ball. 

So I am sort of hoping that you have encountered the same, and made a clever autocommand or something that avoids the breakageness. 

If you have no idea what I'm talking about, then I'm sorry, and i guess you could just ignore me.

**Comment by ldx** [comment]
Hey @unphased,

can you paste that error message? For me, these two plugins work without issues.

**Comment by unphased** [comment]
Well, the indentfinder i had attempts to call `fopen` in Python on a filename that looks like `fugitive:///home/user/<dir>/.git//0/<fileprojectpath>` which clearly doesn't work. 

So I was going to edit indent_finder.py (the fopen is on line 129 in the python file), but then I saw that you seem to have your own fork of indentfinder which is much better than mine (which I cobbled together 3 years ago from the vim.org page -- among other things, your indentfinder has the python code embedded inside it and it makes it easier to manage as a vim plugin), so upon switching to that, my problem has gone away. Thanks!
